src/main/java/vbn/solver/Z3Solver.java

```diff
@@ -103,7 +103,7 @@ public static Expr handleUnaryConstraints(@NonNull Context ctx, @NonNull Map<Str
         return exprToReturn;
     }
 
-    public static void solve(@NonNull State state) {
+    public static List<Symbol> solve(@NonNull State state) {
```

Comment:
```suggestion
    public static List<ConstantSymbol> solve(@NonNull State state) {
```

Could you create a separate parent class for the constant classes

Reply:
Fixed